### ISSUE TITLE
Cuckoo Package : Whitelisting works for DNS but..

### ISSUE THREAD

**Issue by SparkyNZL** [cuckoosandbox/cuckoo#1385]
The white listing works for DNS but it would be great if it also dropped the TCP HTTPS and HOSTs traffic associated with it. so that your network captures are nice and clean :)

**Comment by jbremer** [comment]
Yes, certainly! I'll have to think about a proper solution, though :) Thanks for the request.

**Comment by SparkyNZL** [comment]
NP I was thinking of a Whitelist of domain names and one for IP's then you could control both, so if you wanted some IP traffic and not others , the drive behind this is to make the import of IOC's into MISP a bit smoother and not so high in maintenance

**Comment by jbremer** [comment]
Agreed! There are plenty of use-cases here, but doing it properly certainly requires quite some effort, so let's see when this is prioritized ;-)

**Comment by lillypad** [comment]
Bump 😄
